tests/unit/command/test_run.py

```diff
@@ -2,7 +2,7 @@
 from dvc.command.run import CmdRun
 
 
-def test_run(mocker, dvc_repo):
+def test_run(mocker):
```

Comment:
Removing `dvc_repo` makes these tests run in dvc repo of the core project, which is bad :( We need `dvc` fixture here still.